lib/site_template/Gemfile.erb

```diff
@@ -0,0 +1,24 @@
+source "https://rubygems.org"
+ruby "<%= RUBY_VERSION %>"
+
+# Hello! This is where you manage which Jekyll version is used to run.
+# When you want to use a different version, change it below, save the
+# file and run `bundle install`. Run Jekyll with `bundle exec`, like so:
+#
+#     bundle exec jekyll serve
+#
+# This will help ensure the proper Jekyll version is running.
+# Happy Jekylling!
+gem "jekyll", "~> <%= Jekyll::VERSION %>"
+
+# This is the default theme for new Jekyll sites. You may change this to anything you like.
+gem "minima"
+
+# If you want to use GitHub Pages, remove the "gem "jekyll"" above and
+# uncomment the line below. To upgrade, run `bundle update github-pages`.
+# gem "github-pages", group: :jekyll_plugins
```

Comment:
> And looking at the diff, I don't see that it was originally there...

I understood your point clearly.
This comment is only to show, that those comments were [added months ago](https://github.com/jekyll/jekyll/commit/09f9f193d8e5fb714a6852c9276be1c6e55efbe8).